client/app/visualizations/chart/plotly/prepareLayout.js

```diff
@@ -118,10 +118,10 @@ function prepareBoxLayout(layout, options, data) {
 
 export default function prepareLayout(element, options, data) {
   const layout = {
-    margin: { l: 10, r: 10, b: 10, t: 25, pad: 4 },
+    margin: { l: 10, r: 10, b: 5, t: 20, pad: 4 },
     width: Math.floor(element.offsetWidth),
     height: Math.floor(element.offsetHeight),
-    autosize: true,
+    autosize: false,
```

Comment:
What do we lose by disabling autosize?

Reply:
Built-in autosize feature is based on `window.onresize`, and since we have implemented this feature on our own (via `resizeObserver` service which is even more powerful) - we don't really need it. So  turned it off just to save some resources.